onnxruntime/core/providers/cpu/generator/constant_of_shape_base.h

```diff
@@ -158,6 +158,7 @@ void ConstantOfShapeBase<EnabledOutputTypeList>::SetValueFromTensorProto(const O
     CASE_FETCH_VALUE_DATA(uint16_t)
     CASE_FETCH_VALUE_DATA(uint32_t)
     CASE_FETCH_VALUE_DATA(uint64_t)
+    CASE_FETCH_VALUE_DATA(BFloat16)
```

Comment:
looks we only have opset  9 ConstantOfShape, but opset 20 starts to support BFLOAT16 https://github.com/onnx/onnx/blob/main/docs/Changelog.md#ConstantOfShape-20. 

It might be okay for us to support more than needed in opset 9, while better to add some comments for making it clear.

Reply:
Sure, will add comments, thanks. Our document (https://github.com/microsoft/onnxruntime/blob/main/docs/OperatorKernels.md) claims ConstantOfShape supports bf16 but not in our code which I think is worse